modules/telco-core-logging.adoc

```diff
@@ -1,21 +1,22 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-design-components.adoc
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="telco-core-logging_{context}"]
 = Logging
 
 New in this release::
-* Cluster Logging Operator 6.0 is new in this release.
-Update your existing implementation to adapt to the new version of the API.
+* No reference design updates in this release
 
 Description::
-The Cluster Logging Operator enables collection and shipping of logs off the node for remote archival and analysis. The reference configuration ships audit and infrastructure logs to a remote archive by using Kafka.
+The Cluster Logging Operator enables collection and shipping of logs off the node for remote archival and analysis.
+The reference configuration ships audit and infrastructure logs to a remote archive using Kafka.
```

Comment:
Should the _by_ remain here? 

_archive using Kafka_ suggests an archive that uses Kafka.